packages/backend/src/config/parseConfig.ts

```diff
@@ -211,6 +217,10 @@ const mergeWithEnvironment = (config: LightdashConfigIn): LightdashConfig => {
             environment:
                 process.env.NODE_ENV === 'development' ? 'development' : mode,
         },
+        fullstory: {
+            orgId: process.env.FULLSTORY_ORG_ID || '',
+            devMode: process.env.NODE_ENV === 'development',
```

Comment:
```suggestion
            devMode: process.env.NODE_ENV === 'development || lightdashConfig.mode === LightdashMode.DEMO || lightdashConfig.mode === LightdashMode.PR',
```

Reply:
👍 although as long as we don't set the key in dev/pr then this won't fire anyway!